docs/user/datastores/runtime_config.rst

```diff
@@ -213,6 +213,17 @@ don't intersect the geometry. This behavior can be controlled through two proper
 decomposed into. If set below 2, no decomposition will be performed and instead the geometry envelope will be used.
 Also see ``geomesa.query.decomposition.bits``, above.
 
+geomesa.query.processing.or.threshold
```

Comment:
I've lost track, is this a System property only or can it be passed as query hint?

Reply:
just sys prop